docs/vscode.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Usage with Visual Studio Code
```

Comment:
This may need a more specific title. Would this work?

Another option would be `Run and debug with Visual Studio Code`, per the comment below

```suggestion
title: Debugging with Visual Studio Code
```